apps/host/src/features/notice/use-image-upload.ts

```
@@ -0,0 +1,93 @@
+      URL.revokeObjectURL(target.previewUrl);
+      blobUrlsRef.current.delete(target.previewUrl);
+    }
+
+    setImages((prev) => prev.filter((_, idx) => idx !== indexToRemove));
+  };
+
+  useEffect(() => {
+    const currentBlobUrls = blobUrlsRef.current;
+
+    return () => {
+      currentBlobUrls.forEach((url) => URL.revokeObjectURL(url));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)